### PR TITLE
Added zap skill

### PR DESCRIPTION
**Pull request by DevelopingUnicorn**
Switch Channel on your Enigma2 Box.

### CODE REVIEW

**Comment by patrickjquinn** [comment]
Hey Marcel, thanks for the PR! You seem to have removed the port from the config in this PR `port: 4567`, would you mind reverting that change and re-submitting your PR?

Thank you!

**Comment by DevelopingUnicorn** [comment]
Oh i see sry my bad! I will.

File: config/index.js [hunk 1]
```diff
@@ -2,7 +2,8 @@ const config = {
     openweathermap: {key: 'f08bfa92a064d679ae0fad789a886f66'},
     wolframalpha: {key: 'U7L4VR-K3WJPLK6Y2'},
     newsapi: {key: 'f4504df34ba9432f80ff040a41736518'},
-    port: 4567
+    port: 4567,
+	zap: {ip: '192.168.178.172'}
```
Comment (timstableford) [inline config/index.js]: Indented with tab not spaces.

File: skills/zap/index.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+const http = require('http');
```
Comment (timstableford) [inline skills/zap/index.js]: Lots of tab indents in this file, replace them with spaces. Also our overlord Pat has something against semi-colons :)

**Comment by timstableford** [comment]
Nevermind my review, looks like Pat was on this :+1:

**Comment by patrickjquinn** [comment]
Haha yes i've never been one for standards but if I dont stick to one set of standards for myself, i'll get totally lost in my own project! 

The linting and formatting tool i'm using 'xo' handles all of the indentation and code structure stuff automatically. The output is formatted against the latest ES2015 standards so it has more of a thing against semi-colons than I do ;) 

This way its closer to other modern scripting languages like swift etc and less likely to become antiquated any time soon!

**Pull request closed without merging**